recipes/bullet3/all/bullet_extra.cmake

```diff
@@ -0,0 +1,10 @@
+set(BULLET_INCLUDE_DIR "${Bullet_INCLUDE_DIRS}")
+set(BULLET_INCLUDE_DIRS "${Bullet_INCLUDE_DIRS}")
+set(BULLET_LIBRARY_DIR "${Bullet_LIB_DIRS}")
+set(BULLET_LIBRARY_DIRS "${Bullet_LIB_DIRS}")
+set(BULLET_LIBRARIES "${Bullet_LIBRARIES}")
+set(BULLET_DEFINITIONS "${Bullet_DEFINITIONS}")
+set(BULLET_VERSION_STRING "${Bullet_VERSION}")
+
+set(BULLET_ROOT_DIR "${Bullet_ROOT}")  # FIXME: Bullet_ROOT variable not available
+set(BULLET_USE_FILE "${Bullet_ROOT}/lib/cmake/bullet/UseBullet.cmake")  # FIXME: Bullet_ROOT variable not available
```

Comment:
If the path `lib/cmake/bullet/UseBullet.cmake` is provided by the `self.cpp_info.build_modules` it would be automatically included by the Conan find generators. So for me, this `bullet_extra.cmake` shouldn't be necessary.

Reply:
Update: Looking at `https://github.com/bulletphysics/bullet3/blob/master/UseBullet.cmake` these shouldn't be necessary either. Conan should be able to provide the way to link with your library automatically.

Reply:
By providing this file (and re-using the conan variables), the conan `FindBullet.cmake` file is a drop-in replacement for the upstream bullet config file.
Otherwise, consumers would need special logic for conan.